src/extras/module-types.js

```diff
@@ -26,6 +26,8 @@ import { errMsg } from '../err-msg.js';
         if (!res.ok)
           throw Error(errMsg(7, process.env.SYSTEM_PRODUCTION ? [res.status, res.statusText, url, parent].join(', ') : res.status + ' ' + res.statusText + ', loading ' + url + (parent ? ' from ' + parent : '')));
         var contentType = res.headers.get('content-type');
+        if (!contentType)
+          throw Error(errMsg(4, 'Missing header "Content-Type"'));
```

Comment:
To keep the bundle size down, we only include the error message text in the system.js build, while preferring only the error code and relevant variables for the system.min.js build.

```suggestion
          throw Error(errMsg(4, process.env.SYSTEM_PRODUCTION ? [url, parent] : 'Missing header "Content-Type", loading ' + url + (parent ? ' from ' + parent : '')));
```